src/Data/Bit/F2Poly.hs

```diff
@@ -132,6 +134,9 @@ instance Integral F2Poly where
   divMod = quotRem
   mod = rem
 
+instance Show F2Poly where
+  show = (:) '0' . (:) 'b' . flip (showIntAtBase 2 intToDigit) "" . toInteger
```

Comment:
In presence of `showIntAtBase` it would be more natural to define `showsPrec = const $ ...` instead of `show`.

Reply:
Actually, nevermind, it is fine as is.

Reply:
I will revert it then